guide/src/reference/cli.md

```diff
@@ -9,6 +9,18 @@ always be listed via `wasm-bindgen --help`.
 
 [wasm-pack]: https://github.com/rustwasm/wasm-pack
 
+## Installation
+
+Although the command line tool is available through its own crate, `wasm-bindgen-cli`, the recommended way to install it is through installing the `wasm-pack` crate:
+
+```
+# Recommended:
+cargo install wasm-pack
```

Comment:
This isn't actually the recommended way to install wasm-pack. I think it would be best for it to just link to the [wasm-pack website](https://rustwasm.github.io/wasm-pack/installer/).

Reply:
I think I'll lean on you to fix it from here. I know basically nothing!

Reply:
I would replace it with this:

````
The command line tool is available through its own crate, `wasm-bindgen-cli`.

The recommended way to install it is by using
[`wasm-pack`](https://rustwasm.github.io/wasm-pack/installer/).

However, if you want to install `wasm-bindgen-cli` by itself, you can do this:

```
cargo install wasm-bindgen-cli
```
````